src/dash-playlist-loader.js

```diff
@@ -332,6 +334,7 @@ export default class DashPlaylistLoader extends EventTarget {
     if (!playlist.sidx) {
       // Continue asynchronously if there is no sidx
       // wait one tick to allow haveMaster to run first on a child loader
+      window.clearTimeout(this.mediaRequest_);
```

Comment:
why are we clearing here? seems like this should happen in `haveMaster` and `haveMetadata` where `mediaRequest` is set to `null` currently